package.json

```
@@ -149,7 +150,7 @@
     "test:type": "yarn workspaces foreach -piv --include '@uppy/*' --exclude '@uppy/{angular,react-native,locales,companion,provider-views,robodog,svelte}' exec tsd",
     "test:unit": "yarn run build:lib && jest --env jsdom",
     "test:watch": "jest --env jsdom --watch",
-    "test:size": "size-limit",
+    "test:size": "size-limit --why",
```

nit: you'll might get outdated results if the build is outdated, might be a good idea to add the build command.
```suggestion
    "test:size": "yarn build:lib && size-limit --why",
```

---

Very good point!